mobileapp/src/components/screens/notification/NotificationDetailScreen/NotificationDetailScreen.tsx

```diff
@@ -92,13 +110,63 @@ export const NotificationDetailScreen: React.FC<
     setErrorDialog(false)
   }
 
+  /**
+   * Return View with data from NotificationDetails
+   */
   return (
-    <Center justifyContent="center" h="100%">
+    <Center justifyContent="flex-start" h="100%">
       <LoadingModal processing={processing} />
       <ErrorAlert isOpen={errorDialog} close={alertClose} />
-      <Text>{notificationId}</Text>
-      <Button onPress={confirm}>confirm</Button>
-      <Button onPress={deny}>deny</Button>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.id}
+      </Text>
+      <Text italic fontSize={17}>
+        {details?.id}
+      </Text>
+      <Text fontSize={20} fontWeight="bold">
+        {R.strings.details.type}
+      </Text>
+      <Text italic fontSize={17}>
```

Comment:
Pojedyńczy tekst (nagłówek + zawartość) musi być osobnym komponentem, zbyt wiele razy się powrarza dokładnie to samo.

Reply:
zrobione

Reply:
no tak średnio, bo miał być JEDEN, nie 6 z tym samym html'em